### PR TITLE
ADE-671 = Update nits in scripts index file

### PR DESCRIPTION
**Pull request by shouvSarker**
Just a couple of nits coming from https://stash.agiledigital.com.au/projects/AAA/repos/monitoring/pull-requests/9/overview

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -25,6 +23,7 @@ const directorySearch = (localDirectory: string) =>
  */
 const rootCommand = yargs;
 
+// Allows to find in any scripts in the scripts folder without explicitly stating in the index file.
```
Comment (dspasojevic) [inline src/index.ts:26]: Is this line "finding" or doing something with them?

Comment (shouvSarker) [reply src/index.ts]: Updated.

File: src/index.ts [hunk 2]
```diff
@@ -8,13 +8,11 @@ import yargs from 'yargs';
...
 export type RootCommand = typeof rootCommand;
 
-const javaScriptFileMatch = new RegExp(/^[A-z0-9/-]+\.js$/);
-
 // Only finds scripts in top level
 const directorySearch = (localDirectory: string) =>
   readdirSync(join(__dirname, localDirectory), { withFileTypes: true })
     .map(({ name }) => name)
-    .filter((value) => javaScriptFileMatch.test(value))
+    .filter((value) => value.endsWith('.js'))
```
Comment (SamMurphyDev) [inline src/index.ts:15]: As we are making this change here, the `rolup.config.js` also has a similar function which could be swapped as well

Comment (shouvSarker) [reply src/index.ts:15]: Updated and pushed!

**Review by SamMurphyDev — APPROVED** [review]

**Pull request merged**